conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -38,18 +41,17 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-
+conda install --yes conda-build==1.18.2
```

Comment:
We have dropped this pinning in `master` and it will be gone in the next release.